us-medical-insurance-costs-checkpoint.ipynb

```diff
@@ -18,7 +18,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        print(f"median: {median}")
</pre><p>Super minor nitpick: missing dollar signs in median cost (which should be called median cost just for the benefit of the hypothetical people for whom this is intended)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/laurenashley/US-Medical-Insurance-Costs/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>